newIDE/electron-app/app/main.js

```diff
@@ -140,11 +140,12 @@ app.on("ready", function() {
 
   ipcMain.on(
     "piskel-changes-saved",
-    (event, imageResources, newAnimationName) => {
+    (event, imageResources, newAnimationName, fileMetadata) => {
```

Comment:
Why `fileMetadata`? Being explicit is a good thing, but all the other place are refering only to `metadata` and I'm uneasy with "file" because it's not metadata related to a file right?. More like `allResourcesMetadata` or some other name that conveys that it is the global metadata for all resources passed.

Reply:
Oh ok I see that this object will contain things like singleFrame, data, etc... We should find it another name, there is too much "metadata" everywhere otherwise ;)

Reply:
When we say file Metadata, you are right - we think of an actual file with metadata on it. When I wrote the name I was thinking about data that will generate a  file or files. The more I think about it and you point it out to me- the word metadata doesn't really communicate that goal. Its more of a word of the way we store it - as metadata of a game object.

So how should we name this? ImageEditorData or ExternalEditorData would make much more sense imo. Maybe even better include the name of the editor - jfxrData/piskelData.

What do you think? :)

Reply:
we could call it just fileData ? It is data that is used to generate files basically